pyproject.toml

```diff
@@ -25,6 +25,7 @@ dynamic = [
     "version",
 ]
 dependencies = [
+    "asv-runner>=v0.0.4",
```

Comment:
I thought we were going to change the name to `asv_runner` so that the pip install name is the same as the import.

Reply:
The name of the package is `asv_runner`, but `pip` (or PyPI) converts `_` to `-`. As discussed here:

https://github.com/ros/rosdistro/issues/18116#issuecomment-394897086

Also here: https://stackoverflow.com/a/52828384/1895378

Reply:
I know that this Just Works(tm). But now you have to do `pip install avs-runner` and `import avs_runner`. I thnk the UX would be better if they were unified, no?

Reply:
> I know that this Just Works(tm). But now you have to do `pip install avs-runner` and `import avs_runner`. I thnk the UX would be better if they were unified, no?

Yup, but I can't think of a good naming paradigm to do so, since `asv-runner` is not PEP-8 compliant and with the underscores we get to this situation.

Reply:
I suggest you change the name on PyPI to be PEP-8 compliant: `asv_runner` or `asvrunner` or `uls` or something else, but not `asv-runner`

Reply:
> I suggest you change the name on PyPI to be PEP-8 compliant: `asv_runner` or `asvrunner` or `uls` or something else, but not `asv-runner`

The package name (as per the `pyproject.toml` is `asv_runner`). PyPI swaps `_` for `-`. I was under the impression that most (all?) the plugin based projects used a similar naming convention (`_` but resolves to `-` on PyPI). I could switch to `asvrunner` but it might be best to stick to (and document) one convention, e.g. "use underscores" and let PyPI / `pip` do the renaming they do.

Reply:
Ahh, got it thanks.